package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "4.5.1",
+  "version": "4.5.4",
```

Comment:
Why not 4.5.2?

Reply:
Alot of other PR's 4.5.2, 4.5.3 etc

Reply:
You're going to have to rebase/merge anyway...

Reply:
Yup? What's the problem?

Reply:
No problem, but even though you've pre-set the app version here, you're going to need to update/merge this line again anyway.

Resolving this comment.